README.md

```diff
@@ -1 +1,170 @@
-# investgrow-hub
\ No newline at end of file
+
+<p align = "center"><img src="src\assets\Investgrow-HUB.png"></p>
+
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 React \& Redux Capstone project - Investgrow-HUB ](#-react--redux-capstone-project---investgrow-hub-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 React & Redux Capstone project - Investgrow-HUB <a name="about-project"></a>
+
+**Investgrow-HUB
+Investgrow-HUB is a React and Redux project that leverages real live data from Financial modeling prep API. My mission is to develop a mobile web application that helps people monitor stock prices.
+
+ 📗 [Live Demo] 
+ 
+ - Here is a <a href="https://investgrow-hub.onrender.com">link</a> to my live website.
+ - Here is a <a href="https://www.loom.com/share/cc7d7db3af6d4c08acf33ef6c84f95e3?sid=c998c386-d268-4fd9-94d4-e95a1b27cdba">link</a> to my live presentation video.
```

Comment:
- [ ] Good Job for documenting your project ,  by record a video presenting the features of the project you built.  The content of the video include :a description of the project and a demo of the project features. but you didn't  highlight some interesting piece of code or something you built that you are very proud of.** and also the recorded video miss the audio it's silent kindly make sure that also the video .